master/static/srv/get_task_logs_fields.sql

```diff
@@ -1,6 +1,6 @@
 SELECT
     array_to_json(
-        array_remove(
+        array_replace(
```

Comment:
i think the whole approach here is a little off - we converted everything to strings and used `""` to indicate null.. seems to make a lot more sense _not_ to convert any types and just let null but null, but not filter it?